pallets/anonymity-mining-claims/src/lib.rs

```diff
@@ -135,31 +158,40 @@ pub mod pallet {
 		type ForceOrigin: EnsureOrigin<Self::RuntimeOrigin>;
 	}
 
+	/// The map of trees to the maximum number of anchor edges they can have
+	#[pallet::storage]
+	#[pallet::getter(fn max_edges)]
+	pub type MaxEdges<T: Config<I>, I: 'static = ()> = StorageValue<_, u32, ValueQuery>;
+
 	/// A helper map for denoting whether an tree is bridged to given chain
 	#[pallet::storage]
-	#[pallet::getter(fn linkable_tree_has_edge)]
-	pub type LinkableTreeHasEdge<T: Config<I>, I: 'static = ()> =
-		StorageMap<_, Blake2_128Concat, (T::TreeId, T::ChainId), bool, ValueQuery>;
+	#[pallet::getter(fn registered_resource_ids)]
+	pub type RegisteredResourceIds<T: Config<I>, I: 'static = ()> =
```

Comment:
In theory the anonymity claims could be built on a chain with a `SignatureBridge` which does register resource IDs. We could and maybe should support a post-registration hook on that pallet which we can use to hook in here so we don't have to double register things all the time.